connectors/src/api/get_connector_permissions.ts

```diff
@@ -36,9 +36,16 @@ const _getConnectorPermissions = async (
   if (
     req.query.filterPermission &&
     typeof req.query.filterPermission === "string" &&
-    ["read"].includes(req.query.filterPermission)
+    ["read", "write"].includes(req.query.filterPermission)
   ) {
-    filterPermission = "read";
+    switch (req.query.filterPermission) {
```

Comment:
Why is this switch useful can't we do `filterPermission = req.query.filterPermission` since we already know it's either `"read"` or `"write"` ?

Reply:
typescript refuses

Reply:
Replace the ["read", "write"].includes thing above by `req.query.filterPermission === "read" || req.query.filterPermission === "write"` and I think it should work

Reply:
Removed the if and rely on the switch only

Reply:
We don't handle "read_write" on purpose here ? Or not an accepted value ?

Reply:
> Removed the if and rely on the switch only

Didn't work using the if ?

Reply:
I don't think we have the code to handle `read_write` at the moment